README.adoc

```diff
@@ -324,6 +324,109 @@ X-App-Name: my-system
 Verify that `{inventory-api}/system-service` is working as intended.
 If it is not, then check the configuration of the credentials.
 
+== View and Update ConfigMaps and Secrets
+
+Kubernetes has the `describe` command to show details about any ressources. We can use this to display ConfigMaps and Secrets by running the following:
+[role=command]
+```
+kubectl describe configmaps sys-app-name                                                                                                                                          
+```
+
+The output should be something similar to the following. Notice the key `name` under Data has the value `my-system`:
+[source, role="no_copy"]
+....
+Name:         sys-app-name
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+name:
+----
+my-system
+Events:  <none>
+....
+
+Similarly to view the `Secret`, use the following command:
+[role=command]
+```
+kubectl describe secrets sys-app-credential
+```
+
+The output should be similar to the following. Notice that the values for `password` and `username` are hidden.
+[source, role="no_copy"]
+----
+Name:         sys-app-credentials
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+password:  6 bytes
+username:  3 bytes
+----
+
+You can quickly modify ConfigMaps by running the following command. This sets the key `name` to the value `new-name`. Note that this creates a new ConfigMaps and overwrites the old one. 
+[role=command]
+```
+kubectl create configmap sys-app-name --from-literal name=new-name --dry-run=client --save-config=true -o yaml | kubectl replace -f -  
```

Comment:
Syntax error when running this command

```
Error: invalid argument "client" for "--dry-run" flag: strconv.ParseBool: parsing "client": invalid syntax
```

Reply:
What version of kubectl are you running

Reply:
My kubectl version is as follows:

```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-13T16:04:18Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

Reply:
I see, maybe its a version problem? Mine are as follows:
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-12T01:09:16Z", GoVersion:"go1.15.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-13T16:04:18Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```
Could you try updating kubectl? `brew upgrade kubernetes-cli ` If you have `brew` installed

Reply:
Application works now, guess my version was just outdated